### ISSUE TITLE
Testing against the testThaliMobileNative.js tests that don't use connect.

### ISSUE THREAD

**Issue by baydet** [thaliproject/Thali_CordovaPlugin#1157]

**Comment by baydet** [comment]
blocked untill super branch will be merged into vNext

**Comment by baydet** [comment]
most of the non connect tests from testThaliMobileNative are passing. Currently I'm working rewriting these tests.

**Comment by yaronyg** [comment]
Good news, discovery tests pass!

**Comment by baydet** [comment]
several test that are using `connect` are rewritten to execute on both platforms. Now I'm blocked on implementation of the new thaliMobileNative interface.

**Comment by yaronyg** [comment]
@baydet Which branch is your work in? I'd like to merge with it for the work I'm doing.

**Comment by baydet** [comment]
blocked on thaliproject/Thali_CordovaPlugin#1199

**Comment by yaronyg** [comment]
BTW, why are you blocked? Why don't you just merge Dersim's work into a temporary branch and test from there?

**Comment by baydet** [comment]
Well It's not so easy to work on this issue. Now It is impossible to run tests from iOS_dersim_881 and iOS_1079. On client side I'm receiving these logs after passing Unit tests

https://gist.github.com/baydet/3d1cdf07c8e62278390f09ac1fd7699e

So I'm trying to figure handle this (create some tiny workaround to make coordinated server work again)

**Comment by baydet** [comment]
during testing I faced with zombie advertisers in no peer discovery test "thaliproject/Thali_CordovaPlugin#1323"

Also I received some weird errors for example test `cannot call multiConnect when start listening for advertisements is not active` is failing because 'multiConnectResolved wasn't register to native' error.

**Comment by yaronyg** [comment]
I just updated iOS_1079. Everything is now set to iOS and I have confirmed that testThaliMobileNative.js, testThaliMobileNativeDiscoveryCoordinated.js and testThaliMobileNativeiOS.js all pass the mock without issue.

**Comment by baydet** [comment]
We have these tests failing:

`no peer discovery` thaliproject/Thali_CordovaPlugin#1323 
`Connect port dies if not connected to in time` - thaliproject/Thali_CordovaPlugin#1340 

`Can shift large amounts of data` - about every 3rd run is failing for 3 devices thaliproject/Thali_CordovaPlugin#1342 
`Test updating advertising and parallel data transfer` - constantly failing thaliproject/Thali_CordovaPlugin#1341
